web/templates/settings/notification_settings.hbs

```diff
@@ -65,15 +65,17 @@
         </div>
 
         <div class="input-group">
-            <label for="automatically_follow_topics_policy" class="settings-field-label">{{ settings_label.automatically_follow_topics_policy }}</label>
+            <label for="automatically_follow_topics_policy" class="settings-field-label">
+                {{ settings_label.automatically_follow_topics_policy }}{{> ../help_link_widget link="/help/follow-a-topic" }}</label>
             <select name="automatically_follow_topics_policy" class="setting_automatically_follow_topics_policy prop-element settings_select bootstrap-focus-style"
               id="{{prefix}}automatically_follow_topics_policy" data-setting-widget-type="number">
                 {{> dropdown_options_widget option_values=automatically_follow_topics_policy_values}}
             </select>
         </div>
 
         <div class="input-group">
-            <label for="automatically_unmute_topics_in_muted_streams_policy" class="settings-field-label">{{ settings_label.automatically_unmute_topics_in_muted_streams_policy }}</label>
+            <label for="automatically_unmute_topics_in_muted_streams_policy" class="settings-field-label">
+                {{ settings_label.automatically_unmute_topics_in_muted_streams_policy }}{{> ../help_link_widget link="/help/mute-a-topic" }}</label>
```

Comment:
`git grep help_link_widget` suggests that we do not have a pattern of having these appear without a space between the label and the help widget. You should put it on its own line like the others, the code will be more readable.

Reply:
Thanks,
`            <label for="automatically_follow_topics_policy" class="settings-field-label">
                {{ settings_label.automatically_follow_topics_policy }}
                {{> ../help_link_widget link="/help/follow-a-topic" }}
            </label>
`

adding this